src/main/java/uk/gov/hmcts/ccd/domain/model/aggregated/CaseView.java

```diff
@@ -22,11 +22,11 @@ public void setChannels(String[] channels) {
         this.channels = channels;
     }
 
-    public CaseViewTrigger[] getTriggers() {
+    public CaseViewActionableEvent[] getTriggers() {
         return triggers;
     }
 
-    public void setTriggers(CaseViewTrigger[] triggers) {
+    public void setTriggers(CaseViewActionableEvent[] triggers) {
```

Comment:
better rename also method and parameter

Reply:
renamed to setActionableEvents and getActionableEvents